buttons.css

```diff
@@ -0,0 +1,38 @@
+.loginbutton
+{
+	margin: 25%;
+	padding-left: 40px;
```

Comment:
Now for a challenge: write these padding measurements in a single line, in the least amount of characters.